### ISSUE TITLE
Kane's System Rule 'Order.process_calculateddeliverycharge' caused the following exception: AttributeError("'Order' object has no attribute 'passed_obj'",)

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1400]
_Imported from trac ticket 1400.
Created by **kbower**
Opened in trac: Jan 25, 2012, 9:21:27 AM
Last modified in trac: Jan 25, 2012, 10:59:48 AM_
***
One of the class opened up an order from yesterday and received this error message. Our wireless connection on the laptop also went down at the same time. Not sure which problem came up first.

SO# 0124209OX34 in Train

A Fatal error was encountered when communicating with the server.

```
Error message:
RError: System Rule 'Order.process_calculateddeliverycharge' caused the following exception: AttributeError("'Order' object has no attribute 'passed_obj'",)
Traceback (most recent call last):
  File "/home/rarch/trunk/src/appserver/pylotengine/model/objects.py", line 6489, in system_rule_allow_none
    return params['__rule_fn__']()
  File "Order.process_calculateddeliverycharge", line 9, in __rule_fn__
  File "/home/rarch/trunk/src/appserver/pylotengine/model/objects.py", line 125, in __getattr__
    raise AttributeError("'%s' object has no attribute '%s'" % (self.__class__.__name__, name))
AttributeError: 'Order' object has no attribute 'passed_obj'
```

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 25, 2012, 10:59:48 AM:_

(Changeset [[2218]](../commit/2aaaa11ca6567458a7396271ad4b3ab0a86bf94b))
 fix #1400 (Kane's System Rule 'Order.process_calculateddeliverycharge' caused the following exception: AttributeError("'Order' object has no attribute 'passed_obj'",))

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 25, 2012, 10:59:48 AM:_ **closed**
